tests/test_exact_computer.py

```diff
@@ -79,6 +79,52 @@ def test_exact_computer_on_soum():
         # Assert efficiency for SV
         assert (np.sum(probabilistic_values["SV"].values) - predicted_value) ** 2 < 10e-7
 
+        index = "ELC"
+        # Core only works on normalized games
+        soum_normalized = SOUM(n, n_basis_games=n_basis_games, normalize=True)
+        exact_computer = ExactComputer(n_players=n, game_fun=soum_normalized)
+
+        predicted_value = soum_normalized(np.ones(n))[0]
+
+        egalitarian_least_core = exact_computer.compute_egalitarian_least_core(index)
+        # Assert efficiency for Egalitarian Least Core
+        assert (np.sum(egalitarian_least_core.values) - predicted_value) ** 2 < 10e-7
+
+
+def test_exact_elc_on_normalized_soum():
+    for i in range(20):
+        n = np.random.randint(low=2, high=10)
+        n_basis_games = np.random.randint(low=1, high=100)
+        soum_normalized = SOUM(n, n_basis_games=n_basis_games, normalize=True)
+
+        exact_computer = ExactComputer(n_players=n, game_fun=soum_normalized)
+
+        predicted_value = soum_normalized(np.ones(n))[0]
+
+        index = "ELC"
+        egalitarian_least_core = exact_computer.compute_egalitarian_least_core(index)
+        # Assert efficiency for Egalitarian Least Core
+        assert (np.sum(egalitarian_least_core.values) - predicted_value) ** 2 < 10e-7
+
+
+@pytest.mark.parametrize(
+    "index, order",
+    [("ELC", 1)],
+)
+def test_exact_elc_computer_call(index, order):
```

Comment:
What should the ELC min_order be in the `InteractionValues` object after creation. I would like to add a check to that. I think it should be min_order == 1, right?